build/maccreatebundle

```diff
@@ -1,4 +1,5 @@
-export VERSION=`cat version.h | grep "define STR_FILE_VERSION2" | sed 's/.*STR_FILE_VERSION2 //' `
+#!/usr/bin/env bash
+export VERSION=`grep "define STR_FILE_VERSION2" version.h | cut -f2 -d\"`
```

Comment:
This also removes the double quotes from the VERSION string : they seemed to cause a malformed plist file.